javaslang/generator/Generator.scala

```diff
@@ -2228,6 +2229,76 @@ def generateMainClasses(): Unit = {
       """
     }
   }
+
+  /**
+    * Generator of javaslang.collection.*ArrayType
+    */
```

Comment:
I find indentations important also (though I aligned it this way on purpose).
Will change it.